idp-binding-service/src/main/java/io/mosip/idp/binding/config/SecurityConfig.java

```diff
@@ -53,6 +56,7 @@ protected void configure(HttpSecurity http) throws Exception {
                 .authorizeRequests()
 				.antMatchers(HttpMethod.POST, servletPath + "/send-binding-otp/**").hasAuthority(sendBindingOtpApiScope)
 				.antMatchers(HttpMethod.POST, servletPath + "/wallet-binding/**").hasAuthority(walletBindingApiScope)
+				.antMatchers(HttpMethod.POST, servletPath + "/validate-binding/**").hasAuthority(validateBindingApiScope)
```

Comment:
let's not add validate-binding API here, let's consider this to be open API for now.